validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.engine.lar.quality
+
+import hmda.parser.fi.lar.LarGenerators
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class LarQualityEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with LarGenerators
+    with LarQualityEngine {
+
+  property("A LAR must pass quality checks") {
+    forAll(larGen) { lar =>
+      whenever(lar.id == 2) {
+        checkQuality(lar).isSuccess mustBe true
+      }
+    }
+  }
```

Comment:
The most relevant bits I found in a search are in #82, specifically [here](https://github.com/cfpb/hmda-platform/pull/82#discussion_r54579326). I think we've also discussed it since then in the context of particular edits, but perhaps not in github issues (or not that I found).